cmd/config-seed/res/properties/go.mod

```diff
@@ -1 +1,3 @@
 module github.com/edgexfoundry/edgex-go/cmd/config-seed/res/properties
+
+go 1.12
```

Comment:
are we now building with go 1.12?

Reply:
No. @brandonforster probably has it installed locally. This line should be removed.

Reply:
No mods to this file should be included with this PR.